apps/console/src/features/users/pages/user-edit.tsx

```diff
@@ -83,7 +83,7 @@ const UserEditPage = (): ReactElement => {
 
         getGovernanceConnectors(ServerConfigurationsConstants.USER_ONBOARDING_CONNECTOR_ID)
             .then((response: GovernanceConnectorInterface[]) => {
-                const properties: ConnectorPropertyInterface[] = [ ...connectorProperties ];
+                const properties: ConnectorPropertyInterface[] = connectorProperties ? [ ...connectorProperties ] : [];
```

Comment:
Since we expect `connectorProperties` to be an array, shall we do an array type check here as well?

```suggestion
                const properties: ConnectorPropertyInterface[] = connectorProperties && Array.isArray(connectorProperties)
                    ? [ ...connectorProperties ]
                    : [];
```

Reply:
Resolved by c0bc1c4